### PR TITLE
docs(plugin-meeting): update samples meetings management

### PR DESCRIPTION
**Pull request by jonniespratley**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-293018

Updated ui for meetings management:

- Updated CSS to have styled buttons and inputs, as well as utilities for padding/margin. 
- Updated layout to have fixed streams sidebar.
- Updated create meeting form to handle submissions on enter key
- Updated current meetings list to use radio input.
- Added join button below meetings list to join a meeting that was selected by radio.
- Updated leave meeting logic to leave the meeting of the selected meeting radio input.
- Update wdio test.

### CODE REVIEW

**Review by arun3528 — APPROVED** [review]
LGTM

**Comment by jonniespratley** [comment]
LGTM

File: packages/node_modules/samples/browser-plugin-meetings/index.html [hunk 1]
```diff
@@ -3,438 +3,399 @@
...
+            <fieldset>
+                <legend>Active Meetings</legend>
+                
+            </fieldset>
+          </form>
+    
+          <!-- meetings / local-media -->
+          <fieldset>
+              <legend>Add Local Media</legend>
+              <p class="note">NOTE: This should say what the actions be do..</p>
```
Comment (lalli-flores) [inline packages/node_modules/samples/browser-plugin-meetings/index.html:118]:
```suggestion
              <p class="note">NOTE: In order to send audio/video we must request permissions from the user to access camera and microphone. Once we have access, we should add the corresponding media streams to the meeting</p>
```

File: packages/node_modules/samples/browser-plugin-meetings/index.html [hunk 2]
```diff
@@ -3,438 +3,399 @@
...
+          </fieldset>
+        </section>
+    
+        <section>
+          <h2>Meeting Controls</h2>
+          <p>Once in a meeting utilize these controls to manipulate the meeting.</p>
+          <p>Please note that you must be a meeting moderator in order to lock or record a meeting.</p>
+          <form id="general-controls">
+            <fieldset>
+              <legend>General Controls</legend>
```
Comment (lalli-flores) [inline packages/node_modules/samples/browser-plugin-meetings/index.html:151]:
```suggestion
              <legend>Lock/Unlock</legend>
```

Maybe something to address on the meeting controls PR

**Review by lalli-flores — CHANGES_REQUESTED** [review]
Please make this and other commits with changes to samples of `docs` type. We should not be making an SDK release by changing things in sample apps.

**Review by arun3528 — APPROVED** [review]
Can you rebase

**Review by lalli-flores — APPROVED** [review]

**Comment by lalli-flores** [comment]
Merging with sample tests failing.

- Multi Party Calling sample tests are failing - unrelated to these sample changes
- Call with screen share are consistently failing  - unrelated to these sample changes - would need to be addressed separately

**Pull request merged**